1.8/administration/virtual-networks/ip-per-container.md

```diff
@@ -0,0 +1,194 @@
+                            "name": "dcos",
+                            "prefix": 24,
+                            "subnet": "44.128.0.0/16"
+                        },
+                        "state": {
+                            "status": "STATUS_OK"
+                        },
+                        "subnet": "44.128.1.0/24"
+                    }
+                ]
+            },
+            {
+                "ip": "10.10.0.119",
+                "overlays": [
+                    {
+                        "backend": {
+                            "vxlan": {
+                                "vni": 1024,
+                                "vtep_ip": "198.15.0.3/20",
+                                "vtep_mac": "70:b3:d5:0f:00:03",
+                                "vtep_name": "vtep1024"
+                            }
+                        },
+                        "docker_bridge": {
+                            "ip": "44.128.2.128/25",
+                            "name": "d-dcos"
+                        },
+                        "info": {
+                            "name": "dcos",
+                            "prefix": 24,
+                            "subnet": "44.128.0.0/16"
+                        },
+                        "state": {
+                            "status": "STATUS_OK"
+                        },
+                        "subnet": "44.128.2.0/24"
+                    }
+                ]
+            }
+        ],
+    "network": {
+            "overlays": [
+                {
+                    "name": "dcos",
+                    "prefix": 24,
+                    "subnet": "44.128.0.0/16"
+                }
+            ],
+            "vtep_mac_oui": "70:B3:D5:00:00:00",
+            "vtep_subnet": "198.15.0.0/20"
+        }
+    }
+```
+
+# Deleting Overlay Networks
+
+To delete your overlay network, uninstall DC/OS, then delete the overlay replicated log on the master nodes and the iptable rules on the agent nodes that aer associated with the overlay networks.
+
+## The Replicated Log
+
+DC/OS overlay uses a replicated log to persist the overlay network state across Mesos master reboots and to recover overlay state when a new Mesos master is elected. The overlay replicated log is stored at `/var/lib/dcos/mesos/master/overlay_replicated_log`. The overlay replicated log is **not** removed when DC/OS is uninstalled from the cluster, so you need to delete this log manually before reinstalling DC/OS. Otherwise, the Mesos master will try to reconcile the existing overlay replicated log during startup and will fail if it finds an overlay network that was not configured.
+
+**Note:** The overlay replicated log is different from the [master's replicated log](http://mesos.apache.org/documentation/latest/replicated-log-internals/), which is stored at /var/lib/mesos/master/replicated_log. Removing the *overlay* replicated log will have no effect on the master's recovery semantics.
+
+## iptables
```

Comment:
@sascala, yes you are right. Currently, the expectation is that user needs to perform this step manually. @jdef your point though is valid. Let me raise a JIRA and try and see if this can made part of the uninstall scripts.